docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+* At least 4GB RAM available.
+
+Installing the Client
+=====================
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following in your terminal:
+
+OS X
+----
+
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+------------
+
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+Check the Version
+-----------------
+
+To check the version of Flocker you have installed, run the following:
+
+.. prompt:: bash you@laptop:~$
+
+   flocker-deploy --version
+
+Installing Flocker on Local VMs
+===============================
+
+In this step, you will install two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+
+#. Download the Vagrant configuration file, and the cluster and user credentials:
+
+   .. version-code-block:: console
+
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key
+
+#. Use ``vagrant up`` to start and provision the VMs:
+
+   .. prompt:: bash you@laptop:~$
+
+      vagrant up
+      [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
+      ssh-add ~/.vagrant.d/insecure_private_key
+
+Deploying an Application on the First Host
+==========================================
+
+You will now have the client installed on your local machine, and two instances of Flocker, each on a different host.
+The next step is to create two Docker containers on one of the hosts.
+One container has a Python web application and the other has a Redis server, which stores its data on a volume.
+
+Now you can try our simple tutorial: a Python web application and a Redis server. To begin with you'll need to download our sample yaml files:
+
+.. container:: hidden
+
+   .. Create the files to be downloaded with curl, but don't show download links for them.
+
+   :download:`docker-compose.yml`
+   :download:`deployment-node1.yml`
+   :download:`deployment-node2.yml`
+		 
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/docker-compose.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
+
+
+The :file:`docker-compose.yml` file describes your distributed application (:file:`docker-compose.yml` was formerly known as :file:`fig.yml`):
```

Comment:
> :file:`docker-compose.yml` was formerly known as :file:`fig.yml`

This tutorial hasn't existed before so that isn't the case. I guess that you mean that "Docker Compose was formerly known as Fig". I'm not sure that this is worth mentioning.